staging/src/k8s.io/apiserver/pkg/endpoints/handlers/negotiation/negotiate.go

```diff
@@ -225,7 +226,7 @@ func acceptMediaTypeOptions(params map[string]string, accepts *runtime.Serialize
 		}
 	}
 
-	if options.Convert != nil && !endpoint.AllowsConversion(*options.Convert, accepts.MediaTypeType, accepts.MediaTypeSubType) {
+	if !endpoint.AllowsConversion(options.Convert, accepts.MediaTypeType, accepts.MediaTypeSubType) {
```

Comment:
this is pretty twisty, having `AllowsConversion` check a "no conversion" case

Reply:
AllowsConversionAttribute.  Some now disallow non.

Another wording is AllowMediaType.